tests/phpt/error_handler_respects_error_reporting.phpt

```diff
@@ -51,8 +51,15 @@ $client->getOptions()->setCaptureSilencedErrors(false);
 echo 'Triggering silenced error' . PHP_EOL;
 
 @$b++;
+
+$errorReporting = error_reporting(E_ALL & ~E_WARNING);
+echo 'Triggering silenced by error_reporting error' . PHP_EOL;
```

Comment:
You should move this after the `include` to be sure that no output was done in between.